terraform/cicd-deployment/main.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_cloudformation_stack" "data_exports_child" {
+  # checkov:skip=CKV_AWS_124:SNS topic not required for this use case
+  provider     = aws.destination_account
+  name         = "data-exports-aggregation-child"
+  template_url = "https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/${var.global_values.tag_version}/data-exports-aggregation.yaml"
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"]
+  parameters = {
+    DestinationAccountId = var.global_values.destination_account_id
+    ResourcePrefix       = var.data_exports_child.resource_prefix
+    ManageCUR2           = var.data_exports_child.manage_cur2
+    ManageFOCUS          = var.data_exports_child.manage_focus
+    ManageCOH            = var.data_exports_child.manage_coh
+    SourceAccountIds     = var.global_values.source_account_ids
+    EnableSCAD           = var.data_exports_child.enable_scad
+    RolePath             = var.data_exports_child.role_path
+    TimeGranularity      = var.data_exports_child.time_granularity
+  }
+}
+
+
+resource "aws_cloudformation_stack" "data_exports_management" {
+  # checkov:skip=CKV_AWS_124:SNS topic not required for this use case
+  depends_on = [
+    aws_cloudformation_stack.data_exports_child
+  ]
+  name         = "data-exports-aggregation-mgmt"
+  provider     = aws
+  template_url = "https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/${var.global_values.tag_version}/data-exports-aggregation.yaml"
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"]
+  parameters = {
+    DestinationAccountId = var.global_values.destination_account_id
+    ResourcePrefix       = var.data_exports_management.mgmt_resource_prefix
+    ManageCUR2           = var.data_exports_management.mgmt_manage_cur2
+    ManageFOCUS          = var.data_exports_management.mgmt_manage_focus
+    ManageCOH            = var.data_exports_management.mgmt_manage_coh
+    SourceAccountIds     = var.global_values.source_account_ids
+    EnableSCAD           = var.data_exports_management.mgmt_enable_scad
+    RolePath             = var.data_exports_management.mgmt_role_path
+    TimeGranularity      = var.data_exports_management.mgmt_time_granularity
+  }
+}
+
+
+resource "aws_cloudformation_stack" "cudos_dashboard" {
```

Comment:
these are called foundational stack not cudos_dashboard, these include other dashboards as well